config/v1/types_cluster_version.go

```diff
@@ -41,12 +41,39 @@ type ClusterVersionSpec struct {
 	ClusterID ClusterID `json:"clusterID"`
 
 	// desiredUpdate is an optional field that indicates the desired value of
-	// the cluster version. Setting this value will trigger an upgrade (if
-	// the current version does not match the desired version). The set of
-	// recommended update values is listed as part of available updates in
-	// status, and setting values outside that range may cause the upgrade
-	// to fail. You may specify the version field without setting image if
-	// an update exists with that version in the availableUpdates or history.
+	// the cluster version. Setting this value will trigger an upgrade if
+	// the current architecture or version does not match the desired
+	// architecture or version. The set of recommended update values is
+	// listed as part of available updates in status, and setting values
+	// outside that range may cause the upgrade to fail.
+	//
+	// architecture is an optional field that indicates the desired
+	// value of the cluster architecture. In this context cluster
+	// architecture means either a single architecture or a multi
+	// architecture. architecture can only be set to multi thereby
+	// only allowing updates from single to multi architecture. If
+	// architecture is set version cannot be set and defaults to
+	// the current target release's version. The empty value
+	// indicates that architecture is not set in which case
+	// architecture defaults to the current target release's
+	// architecture.
+	//
+	// version is a semantic versioning identifying the update version.
+	// version is optional if image is specified. version is not allowed
+	// to be specified if architecture is specified and will default to
+	// the current target release's version.
+	//
+	// image is a container image location that contains the update.
+	// image is optional if version is specified and an update exists
+	// with that version in the availableUpdates or history. image is
+	// also optional if architecture is specified.
+	//
+	// force allows an administrator to update to an image that has failed
```

Comment:
`force` is orthogonal to the other `Update` properties, so I don't think we need to discuss it here.

Reply:
Based on your google doc updates you removed the other property descriptions and left `force` in so that's what I've done.

Reply:
removed `force`